app/controllers/questions_controller.rb

```diff
@@ -70,11 +183,32 @@ def log_time
   end
 
   def next
-    setup = Setup.take
-    week = setup.weeks.find_by_number(params[:week_number])
-    RecommenderSystem::Recommender.setup(current_user.id,week.id)
-    best = RecommenderSystem::HybridRecommender.new.get_best
-    los = LearningObject.find(best[0])
-    redirect_to action: "show", id: los.url_name
```

Comment:
Kedze sa vzdy vybera jedna otazka z 10, ktore su v danom teste, tak si myslim, ze je zbytocne tam este nieco odpurucat, ze staci mu dat nahodne  jednu z tych 10 otazok. Ale ak si myslite, ze by bolo vhodne mu ich odporucat, tak to tam kludne mozem dat :).